DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/_FaqCard.cshtml

```diff
@@ -0,0 +1,61 @@
+<div class="searchable-element nhsuk-expander-group word-break" id="@Model.FaqId-card">
+
+  <details class="nhsuk-details nhsuk-expander">
+    <div class="nhsuk-details__text">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-one-quarter">
+          <div class="nhsuk-tag tag card-tag--grey">
```

Comment:
Think we can replace the class here (and for the other tag) with
`nhsuk-tag nhsuk-tag--grey`
which just makes the tags look a bit nice. Think there was some feedback ages ago that tags that look like they currently do are too similar to buttons.

Though these will eventually be replaced with filterable tags anyway, so up to you if you think there is no point.